rules/windows/c2_installutil_beacon.toml

```diff
@@ -18,7 +18,7 @@ name = "InstallUtil Process Making Network Connections"
 risk_score = 21
 rule_id = "a13167f1-eec2-4015-9631-1fee60406dcf"
 severity = "medium"
-tags = ["Elastic", "Windows"]
+tags = ["Elastic", "Host", "Windows", "Advanced Threat Detection", "Defense Evasion"]
```

Comment:
i'm okay with "Host" as tag, but we already have ATT&CK information inside of threat. duplicating it here will be error prone and will almost definitely drift over time.

Reply:
It's also duplicated in the file naming convention which is equally susceptible to drift.  Ideally most all of the tags in this PR can be found in some UI searchable/filterable schema in a future release, and we can eliminate the greater portion of these tags as those will satisfy the desired use case.

Reply:
Drift is definitely a concern. After this merges I can add a unit test to to ensure the "primary tactic" stays aligned with this - similar to https://github.com/elastic/detection-rules/pull/398. There are some other opportunities for unit tests around tag alignment as well.